### DISCUSSION TITLE
프론트에서 요청한 내용

### DISCUSSION THREAD

**Discussion by locodingve** [prgrms-web-devcourse/slam-server#127] [General]
프론트에서 요청한 내용을 적는 공간입니다! 자유롭게 의견 나눠보아요

**Comment by locodingve** [comment]
dto에 담긴 id 타입을 일괄적으로 string 으로 변환하면 어떨까요~?

이유 1. 개발생산성을 위해서
이유 2. 프론트 int 타입 범위 넘을 경우 일어날 이슈 방지

관련 프론트 pr : https://github.com/slamapp/client/pull/70

**Reply by sds1vrk** [reply]
String으로 변경하기로 했습니다. 줄때, 받을때 둘다 String으로 변경하기로 했습니다!

**Reply by locodingve** [reply]
오! 
넵~ 알겠습니다. 👍

**Reply by KwonYeKyeong** [reply]
id타입을 string으로 변경하는 이 부분은 이번 api 수정시 반영되는걸로 결정됐습니다!!

**Reply by locodingve** [reply]
넵~ 알겠습니다~ 👍

**Reply by manudeli** [reply]
감사합니다~!

**Comment by manudeli** [comment]
관련 pull request 남깁니다
https://github.com/slamapp/client/pull/70
